typing/env.ml

```diff
@@ -1616,6 +1635,11 @@ and components_of_module_maker (env, sub, path, mty) =
             let decl' = Subst.type_declaration sub decl in
             let constructors =
               List.map snd (Datarepr.constructors_of_type path decl') in
+            let ext_constructors =
```

Comment:
I probably miss something obvious, but doesn't `Datarepr.constructors_of_type` return an empty list for extensible types (and thus ext_constructors is always empty)?